src/editor/services/EditorService.ts

```diff
@@ -26,42 +26,40 @@ export default class EditorService {
   public getFixedBody = (editor, externalHtml?) => {
     // converting from html to md to html cleans the body
     const bodyHtml = externalHtml ? externalHtml : editor.serialize().body.value;
-
-    let $bodyHtml = $(bodyHtml);
+    const $bodyHtml = $(bodyHtml);
 
     // the html is to replace the body in the editor
     let _fixedBody = '';
 
-    $bodyHtml = $bodyHtml.map(i => {
+    $bodyHtml.map(i => {
```

Comment:
Please use an explicit name for the variable instead of i. `i` stands for index.

Reply:
it is irrelevant